pipeline/notifications/check_message.py

```diff
@@ -3,23 +3,30 @@
 import prefect
 from prefect import task
 
-from pipeline.types import ChatData, NotificationInfo, User
+from pipeline.types import NotificationInfo, NotificationInput
 
 
 class NewMessageNotifier:
```

Comment:
You can inherit the parent class like this:

```python
class NewMessageNotifier(Notifier):
```

Don't forget to import `Notifier`.